pkg/channel/grpc/grpc_channel_test.go

```diff
@@ -36,7 +51,7 @@ func TestInvokeMethod(t *testing.T) {
 	var opts []grpc.DialOption
 	opts = append(opts, grpc.WithInsecure())
 	conn, err := grpc.Dial("localhost:9998", opts...)
-	defer close(t, conn)
+	defer klose(t, conn)
```

Comment:
`close(t, conn)` was shadowing the built in `close(channel)`. For that reason, renamed to `klose`. Can change to something else if needed.

Reply:
on Kubernetes, you klose things.

Probably a typo?

Reply:
> `close(t, conn)` was shadowing the built in `close(channel)`. For that reason, renamed to `klose`. Can change to something else if needed.

Should probably be `closeX` where X describes what's being closed, like `closeConnection`. Otherwise this just looks like a typo.

Reply:
will change